integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/CallMediatorReplyNoDefaultContentTypeTestProxy.xml

```diff
@@ -0,0 +1,15 @@
+<proxy name="CallMediatorReplyNoDefaultContentTypeTestProxy"
+       startOnLoad="true" trace="disable" transports="http https">
+<description/>
+<target>
+    <inSequence>
+        <log level="custom">
+            <property expression="get-property('transport','Content-Type')" name="Default_ContentType_Test_ContentType"/>
```

Comment:
Could you please explain what is done by this statement? 
Should use $ctx::

Reply:
In here, We need to assign the value of the Content-Type header to Default_ContentType_Test_ContentType the exact property. Hence using get-property() rather than $ctx::